### PR TITLE
chore: lock guzlehttp

### PR DESCRIPTION
**Pull request by ibrito182**

### CODE REVIEW

**Review by Felynx3 — APPROVED** [review]

**Review by rgdevment — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
         "symfony/process": "~3|~4|~5",
         "jeremeamia/superclosure": "~2",
         "lstrojny/hmmmath": ">=0.5.0",
-        "guzzlehttp/guzzle": "^6.3|~7",
+        "guzzlehttp/guzzle": "6.5.7",
```
Comment (rgdevment) [inline composer.json]: please set 6.5 or higher

Comment (ibrito182) [reply composer.json]: the next jump right to the v7 :(

Comment (rgdevment) [reply composer.json]: it is not good to limit it to a security fix, on the way, even today another may appear.

Comment (ibrito182) [reply composer.json]: updated

**Review by Felynx3 — APPROVED** [review]

**Review by Felynx3 — APPROVED** [review]

**Review by Felynx3 — APPROVED** [review]

**Review by tavofuentes — APPROVED** [review]

**Review by bpanatta — APPROVED** [review]

**Pull request merged**